### PR TITLE
Change Person class to Day class

### PR DESCRIPTION
**Pull request by ethanso56**
changed variable names too

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any day obtaining a copy
```
Comment (wayne987) [inline LICENSE:5]: i think the person here is not the same person as the one in the model

Comment (ethanso56) [reply LICENSE:5]: oh ya

File: LICENSE [hunk 2]
```diff
@@ -6,7 +6,7 @@ Permission is hereby granted, free of charge, to any person obtaining a copy
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit days to whom the Software is
```
Comment (junhui-phoon) [inline LICENSE:9]: should be the original persons, this MIT license file no need to edit.

**Review by junhui-phoon — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 2 above)
Comment (ethanso56) [reply LICENSE]: done

**Review by junhui-phoon — APPROVED** [review]
LGTM

**Pull request merged**